docs/sources/configuration/integrations/gcp-exporter-config.md

```diff
@@ -0,0 +1,184 @@
+---
+aliases:
+- /docs/agent/latest/configuration/integrations/gcp-exporter-config/
+title: gcp_exporter_config
+---
+
+# gcp_exporter_config
+
+## Overview
+The `gcp_exporter_config` block configures the `gcp_exporter` integration, which is an embedded version of
+[`stackdriver_exporter`](https://github.com/prometheus-community/stackdriver_exporter). This allows for the collection of 
+metrics data from [GCP Cloud Monitoring (formerly stackdriver)](https://cloud.google.com/monitoring/docs).
+
+The exporter supports all metrics available via [GCP's monitoring API](https://cloud.google.com/monitoring/api/metrics_gcp). Metrics
+exporter by this integration will follow the template `stackdriver_<monitored_resource>_<metric_type_prefix>_<metric_type>`. 
+
+As an example,
+for this load balancing metric
+![gcp-exporter-config-metric](gcp-exporter-config-metric.png)
```

Comment:
This image isn't showing up in my preview.  Is it really needed?  Could you link to the documentation topic rather than taking a screenshot of it?   If you decide to keep the screenshot, here are guidelines: https://grafana.com/docs/writers-toolkit/writing-guide/image-guidelines/.  Here's the section on where to store assets: https://grafana.com/docs/writers-toolkit/writing-guide/image-guidelines/#where-to-store-media-assets.  You're supposed to use the asset uploader application and not store images locally.

Reply:
I uploaded it let me know if it's helpful or should be removed